query/doc.html

```diff
@@ -206,21 +233,21 @@ <h4>Result Descriptions</h4>
         <section id="docTerminology">
           <h4>Terminology</h4>
           <p>The SPARQL language includes IRIs, a subset of RDF URI References that omits spaces. Note that all IRIs in SPARQL queries are absolute; they may or may not include a fragment identifier
-          [<a href="#rfc3987">RFC3987</a>, section 3.1]. IRIs include URIs [<a href="#rfc3986">RFC3986</a>] and URLs. The abbreviated forms (<a href="#QSynIRI">relative IRIs and prefixed names</a>)
+          [[RFC3987], section 3.1. IRIs include URIs [[RFC3986]] and URLs. The abbreviated forms (<a href="#QSynIRI">relative IRIs and prefixed names</a>)
           in the SPARQL syntax are resolved to produce absolute IRIs.</p>
-          <p>The following terms are defined in <a class="norm" href="http://www.w3.org/TR/2004/REC-rdf-concepts-20040210/">RDF Concepts and Abstract Syntax</a> <a href="#CONCEPTS">[CONCEPTS]</a> and
+          <p>The following terms are defined in [[[RDF11-CONCEPTS]]] [[RDF11-CONCEPTS]] and
           used in SPARQL:</p>
           <ul>
-            <li><a class="type IRI" href="http://www.w3.org/TR/2004/REC-rdf-concepts-20040210/#dfn-URI-reference">IRI</a> (corresponds to the Concepts and Abstract Syntax term "<code>RDF URI
+            <li><a>IRI</a> (corresponds to the Concepts and Abstract Syntax term "<code>RDF URI
             reference</code>")</li>
-            <li><a class="type literal" href="http://www.w3.org/TR/2004/REC-rdf-concepts-20040210/#dfn-literal">literal</a></li>
-            <li><a class="type lexicalForm" href="http://www.w3.org/TR/rdf-concepts/#dfn-lexical-form">lexical form</a></li>
+            <li><a>literal</a></li>
```

Comment:
This is removing the links.

I hope we can link to the RDF 1.2 - a way to remember to do that is leave the old links in.

Reply:
I left those for "plain literal" and "datatyped literal", as 1.1 doesn't have those concepts.

I removed the explicit linking, as ReSpec now does this through automatic cross-referencing (when standardized definition fragments are used). In the config, "xref" lists the specs for which to look for such definitions; it's includes "RDF11-CONCEPTS", and can eventually change to "RDF12-CONCEPETS", after it's been published. My guess is that "web-platform" is there for boilerplate? We don't use any definitions from there.

Reply:
I found I needed to go back to the `data-cite=“RDF12-CONCEPTS#iri”` style because pulling them in automatically from 1.1 caused some odd conflicts.